review.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+#
+# AWS Lambda Code Grabber
+# v1.0 - TJ
+#
+# Grab all Lambda code bases in one mighty swoop
+# Yes yes, it is just a wrapper around aws :)
+#
+import os
+import sys
+import subprocess
+import errno
+import json
+import argparse
+
+# Requirements
+AWS="/usr/local/bin/aws"
+JQ="/usr/bin/jq"
+
+# Define the directory that's going to store all this
+FUNDIR="functions"
+
+
+
+##### Get awf my land  ######
+
+def banner():
+    print("""
+           __  _
+       .-.'  `; `-._  __  _
+      (_,         .-:'  `; `-._
+    ,'o"(        (_,           )
+   (__,-'      ,'o"(            )>
+      (       (__,-'            )
+       `-'._.--._(             )
+          |||  |||`-'._.--._.-'
+                     |||  |||
+
+             \033[0;32mLambshank v1.0 
+       Lambda Functions Downloader\033[0m
+
+
+""")
+
+
+def setvars(profile, region):
+    if profile is not None:
+        profile = " --profile " + profile
+    else:
+        profile = ""
+    if region is not None:
+        region = " --region " + region
+    return(profile,region)
+
+def createbase():
+    print("Making Functions Directory: " + FUNDIR)
+    try:
+        os.mkdir(FUNDIR)
+    except OSError as exc:
+        if exc.errno == errno.EEXIST:
```

Comment:
you might just be able to catch the EEXIST directly instead of OSError https://docs.python.org/3/library/exceptions.html#FileExistsError